app/scripts/deploy.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+# Очистка старых контейнеров и образов birdlense (только!)
+docker-compose down
+# Удаляем старые образы birdlense, если они есть
+old_images=$(docker images -q birdlense 2>/dev/null)
+[ -n "$old_images" ] && docker rmi $old_images || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)